docs/2 Run a Blockchain/2 Starport command reference.md

```diff
@@ -0,0 +1,30 @@
+# Starport command reference
```

Comment:
I think we shouldn't be hard-coding sections like this into the docs. This will live in the binary itself and at some point, we'll make it possible for the CLI to print out documentation about itself.

The static documentation acts as articles about how Starport CLI works, rather than being a reference.

In my opinion, in a reference you tell what each part of the program does, in the docs, you tell how these parts fit together.